### PR TITLE
Some error message cleanup

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

**Describe what this PR does**
Provide some context for the reviewer

**Is there anything that requires special attention?**
Do you have any questions? Did you do something clever?

**Related issues:**
Mention any github issues relevant to this PR

### CODE REVIEW

File: pkg/glusterfs/controllerserver.go [hunk 1]
```diff
@@ -128,7 +128,7 @@ func (cs *ControllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 		},
 	}
 
-	glog.V(4).Infof("CSI Volume response: %+v", resp)
+	glog.V(4).Infof("CSI volume response: %+v", resp)
```
[the remark this answers is not in the document]
Comment (humblec) [reply pkg/glusterfs/controllerserver.go]: @Madhu-1 its kind of deliberate. it allows us to hide the response in general case. the volume response may container details about the gluster cluster, so hided in first place.

File: pkg/glusterfs/controllerserver.go [hunk 2]
```diff
@@ -342,7 +342,7 @@ func (cs *ControllerServer) ValidateVolumeCapabilities(ctx context.Context, req
 	resp := &csi.ValidateVolumeCapabilitiesResponse{
 		Supported: capSupport,
 	}
-	glog.V(1).Infof("GlusterFS CSI driver volume capabilities: %v", resp)
+	glog.V(1).Infof("glusterfs CSI driver volume capabilities: %v", resp)
```
[the remark this answers is not in the document]
Comment (humblec) [reply pkg/glusterfs/controllerserver.go]: yes, moving to +v

**Comment by humblec** [comment]
@Madhu-1 PTAL.

File: pkg/glusterfs/controllerserver.go [hunk 3]
```diff
@@ -342,7 +342,7 @@ func (cs *ControllerServer) ValidateVolumeCapabilities(ctx context.Context, req
 	resp := &csi.ValidateVolumeCapabilitiesResponse{
 		Supported: capSupport,
 	}
-	glog.V(1).Infof("GlusterFS CSI driver volume capabilities: %v", resp)
+	glog.V(1).Infof("glusterfs CSI driver volume capabilities: %+v", resp)
```
Comment (JohnStrunk) [inline pkg/glusterfs/controllerserver.go]: Probably want to keep "GlusterFS" as a proper name to be consistent w/ usage like L196.
... also present a couple other places in this commit

Comment (humblec) [reply pkg/glusterfs/controllerserver.go]: @JohnStrunk done. Replaced to GlusterFS

**Comment by Madhu-1** [comment]
LGTM

**Comment by humblec** [comment]
Thanks @JohnStrunk & @Madhu-1 . Merging this PR based on 2 approvals.

**Comment by humblec** [comment]
Thanks @JohnStrunk & @Madhu-1 . Merging this PR based on 2 approvals.

**Comment by humblec** [comment]
Thanks @JohnStrunk & @Madhu-1 . Merging this PR based on 2 approvals.

**Comment by humblec** [comment]
Thanks @JohnStrunk & @Madhu-1 . Merging this PR based on 2 approvals.

**Comment by humblec** [comment]
Thanks @JohnStrunk & @Madhu-1 . Merging this PR based on 2 approvals.

**Comment by humblec** [comment]
Thanks @JohnStrunk & @Madhu-1 . Merging this PR based on 2 approvals.

**Pull request merged**